thyrel-web/src/components/room/start/SentenceForm.tsx

```diff
@@ -42,6 +42,13 @@ export default function StartForm() {
       : setSentence(e.target.value);
   }
 
+  function handleKeyPress(e: any) {
```

Comment:
```suggestion
  function handleKeyPress(e: React.KeyboardEvent<HTMLInputElement>) {
```
tu peux trouvé le type ici 
<img width="469" alt="Capture d’écran 2021-05-25 à 14 09 00" src="https://user-images.githubusercontent.com/56388157/119495523-c6854700-bd62-11eb-93e9-27a5980d496d.png">